types/schemas/prototypeApplication/data/Applicant.ts

```diff
@@ -22,124 +27,95 @@ export type BaseApplicant = ContactDetails & {
    * @description Address information for the applicant
    */
   address: UserAddress;
-  /**
-   * @description Contact information for the site visit
-   */
-  siteContact: SiteContact;
 };
 
-export interface Agent extends BaseApplicant {
+/**
+ * @title Applicant with agent
+ * @description Details about the applicant and the agent or proxy applying on their behalf
+ */
+export type ApplicantWithAgent = BaseApplicant & {
```

Comment:
Interactive docs have much better built-in support for displaying titles when exported as a `type`, rather than `interface`, as far as I can tell